src/middleware/header.js

```diff
@@ -240,15 +238,29 @@ module.exports = function (middleware) {
 			function (data, next) {
 				async.parallel({
 					scripts: async.apply(plugins.fireHook, 'filter:scripts.get', []),
+					timeagoLocale: (next) => {
+						languages.list(function (err, languages) {
+							if (err || !languages.some(obj => obj.code === res.locals.config.userLang)) {
+								return next(err);
+							}
+
+							const pathToLocaleFile = '/vendor/jquery/timeago/locales/jquery.timeago.' + utils.userLangToTimeagoCode(res.locals.config.userLang) + '.js';
```

Comment:
```
var localeFilename = 'jquery.timeago.' + userLang + '.js';
path.join(__dirname, '../../public/vendor/jquery/timeago/locales', pathToLocaleFile)
```

Reply:
I use `pathToLocaleFile` twice to refer to the paths both client-side and server-side, that's why it is the way it is.